### ISSUE TITLE
Question on GCAM v5.4 - Direct Air Capture

### ISSUE THREAD

**Issue by atmos-project** [JGCRI/gcam-core#153]
Hello GCAM developers,

Sorry if I miss this on the documentation but how do I turn the Direct Air Capture option on? I have set GCAM v5.4 up to work on a Windows machine but I would like to include the "Direct Air Capture" option.

Thanks

**Comment by mbins** [comment]
In the input/gcamdata/xml folder you will find five XMLs related to DAC (one for each SSP).  GCAM uses SSP2 ("middle of the road") assumptions by default.  If you include dac_ssp2.xml (or one of the other dac_ssp xml files) to your configuration, you are running with DAC.  Note that, as with CCS, DAC will not deploy in scenarios without a carbon price (e.g. the reference scenario).

**Comment by atmos-project** [comment]
Thanks for the reply.

**Comment by atmos-project** [comment]
Sorry to reopen.

I would like to run GCAM v5.4 with DAC but for only one specific region not globally. How can I achieve that?

I have attempted to modify the dac_ssp2.xml by removing all regions leaving only my region of interest. However, the results show no difference relative to a run without DAC.

I would appreciate any suggestion. Thanks

**Comment by jayfuhrman** [comment]
@atmos-project can you post your modified dac input file?

**Comment by atmos-project** [comment]
Thanks for the reply.

I was not able to upload the .xml file (I have received an error message saying 'We don't support that file type'.

I am pasting here the beginning of the file (I have not changed anything in the global-technology-database)...

<?xml version="1.0" encoding="UTF-8"?><scenario>
    <world>
        <region name="Saudi Arabia">
            <supplysector name="CO2 removal">
                <relative-cost-logit>
                    <logit-exponent fillout="1" year="1975">-3</logit-exponent>
                </relative-cost-logit>
                <output-unit>Mt</output-unit>
                <input-unit>EJ or Mt</input-unit>
                <price-unit>1975$/kg</price-unit>
                <keyword final-energy="industry"/>
                <subsector name="dac">
                    <absolute-cost-logit>
                        <logit-exponent fillout="1" year="1975">-0.036</logit-exponent>
                    </absolute-cost-logit>
                    <share-weight fillout="1" year="1975">1</share-weight>
                    <interpolation-rule apply-to="share-weight" from-year="2015" to-year="2100">
                        <interpolation-function name="fixed"/>
                    </interpolation-rule>
                    <stub-technology name="hightemp DAC NG"/>
                    <stub-technology name="hightemp DAC elec"/>
                    <stub-technology name="lowtemp DAC heatpump"/>
                    <stub-technology name="no DAC"/>
                </subsector>
            </supplysector>
            <supplysector name="process heat dac">
                <relative-cost-logit>
                    <logit-exponent fillout="1" year="1975">-3</logit-exponent>
                </relative-cost-logit>
                <output-unit>EJ</output-unit>
                <input-unit>EJ</input-unit>
                <price-unit>1975$/GJ</price-unit>
                <keyword final-energy="industry"/>
                <subsector name="gas CCS">
                    <relative-cost-logit>
                        <logit-exponent fillout="1" year="1975">-6</logit-exponent>
                    </relative-cost-logit>
                    <share-weight fillout="1" year="1975">1</share-weight>
                    <interpolation-rule apply-to="share-weight" from-year="2015" to-year="2100">
                        <interpolation-function name="fixed"/>
                    </interpolation-rule>
                    <stub-technology name="gas CCS"/>
                </subsector>
            </supplysector>
            <supplysector name="airCO2">
                <relative-cost-logit>
                    <logit-exponent fillout="1" year="1975">-3</logit-exponent>
                </relative-cost-logit>
                <output-unit>Mt</output-unit>
                <input-unit>EJ or Mt</input-unit>
                <price-unit>1975$/kg</price-unit>
                <keyword final-energy="industry"/>
                <subsector name="airCO2">
                    <relative-cost-logit>
                        <logit-exponent fillout="1" year="1975">-6</logit-exponent>
                    </relative-cost-logit>
                    <share-weight fillout="1" year="1975">1</share-weight>
                    <stub-technology name="airCO2"/>
                </subsector>
            </supplysector>
            <energy-final-demand name="CO2 removal">
                <perCapitaBased>0</perCapitaBased>
                <final-energy-consumer/>
                <price-elasticity year="2020">0</price-elasticity>
                <price-elasticity year="2025">0</price-elasticity>
                <price-elasticity year="2030">0</price-elasticity>
                <price-elasticity year="2035">0</price-elasticity>
                <price-elasticity year="2040">0</price-elasticity>
                <price-elasticity year="2045">0</price-elasticity>
                <price-elasticity year="2050">0</price-elasticity>
                <price-elasticity year="2055">0</price-elasticity>
                <price-elasticity year="2060">0</price-elasticity>
                <price-elasticity year="2065">0</price-elasticity>
                <price-elasticity year="2070">0</price-elasticity>
                <price-elasticity year="2075">0</price-elasticity>
                <price-elasticity year="2080">0</price-elasticity>
                <price-elasticity year="2085">0</price-elasticity>
                <price-elasticity year="2090">0</price-elasticity>
                <price-elasticity year="2095">0</price-elasticity>
                <price-elasticity year="2100">0</price-elasticity>
            </energy-final-demand>
        </region>
        <global-technology-database>

**Comment by jayfuhrman** [comment]
Does your version of GCAM have a Saudi Arabia region broken out?  In the core version Saudi Arabia is part of the Middle East region, so it's possible the model is just not finding a region called Saudi Arabia to deploy DAC in, or there isn't a CO2 price established for that region if it does exist.

**Comment by atmos-project** [comment]
Yes my GCAM version has Saudi Arabia broken out. Thanks

**Comment by pkyle** [comment]
Just comparing with a local run with DAC, the prices don't look particularly alarming, though my run doesn't have negative prices for CO2 removal. Are you sure that this run didn't deploy DAC? If you print out the "outputs by technology", there should be 4 technologies in the CO2 removal sector, and three of them are DAC.

**Comment by atmos-project** [comment]
Thank for the reply. No DAC is not being deployed. I do not see the 'CO2 removal sector' for Saudi Arabia. I have made a comparison with another run that deploys dac globally. For any region I check the "outputs by technology" query I can see the 4 technologies in the in the CO2 removal sector as you mention, except for Saudi Arabia ...

**Comment by atmos-project** [comment]
Yes, it is defined. It is equal to 1 in Saudi Arabia like in the other regions ...
